Yearn-agents/InstantWithdrawl/src/agent.spec.ts

```diff
@@ -0,0 +1,56 @@
+jest.useFakeTimers();
+import { HandleBlock } from "forta-agent";
+import { TestBlockEvent } from "forta-agent-tools";
+import agent from "./agent";
+import { buildWeb3, build_Mock } from "./contract.mock";
+import { generateFinding } from "./utils";
+import mockAxios from "./jest.mock";
+
+describe("Yearn: Instant Withdraw Agent", () => {
+  let handleBlock: HandleBlock;
+
+  beforeEach(async () => {
+    handleBlock = await agent.providerHandleBlock(
+      buildWeb3() as any,
+      mockAxios
+    );
+  });
+
+  describe("Finding if withdraw possible", () => {
+    it("Account withdraws: If the withdraw is working, dont return any findings", async () => {
+      const blockEvent = new TestBlockEvent();
+
+      const findings = await handleBlock(blockEvent);
+
+      expect(findings).toStrictEqual([]);
+    });
+
+    it("Another account withdraws: If the withdraw is not working, return an alert", async () => {
+      const blockEvent = new TestBlockEvent();
+
+      const findings = await handleBlock(blockEvent);
+
+      expect(findings).toStrictEqual([
```

Comment:
I know that maybe your mocks have all the information to let this test works as expected. I think this is the same issue that we discuss multiple times. The thing is that test should be self-descriptive, this test look exactly the same as the above one and is expecting a finding. Then as you can guess, finding how this test works requires not only look at the previous tests but also look at the mocks and maybe other things. Self-descriptive test are always better.

Reply:
I can rewrite a more descriptive message sure. Otherwise the logic is correct.

Reply:
No, this time is not the same issue, is related with the old discussion, but in this case is the tests suite is not self-descriptive. The `agent.spec.ts` file itself doesn't have any information (in code) about what is being tested, it just have a label in each test.

Reply:
I dont get what you are trying to say.